packages/account-v2/src/containers/POOForm/FileUploadField.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ChangeEvent, useRef, useState } from 'react';
+import clsx from 'clsx';
+import { Field, FieldProps, FormikErrors, useFormikContext } from 'formik';
+import { StandaloneXmarkRegularIcon } from '@deriv/quill-icons';
+import { Button, Input } from '@deriv-com/ui';
+import { TPaymentMethod, TProofOfOwnershipFormValue } from 'src/types';
+import { compressImageFiles, TFile } from 'src/utils';
+
+type TFileUploadFieldProps = {
+    methodId: number | string;
+    paymentMethod: TPaymentMethod;
+    subIndex: number | string;
+};
+
+export const FileUploaderField = ({ methodId, paymentMethod, subIndex }: TFileUploadFieldProps) => {
+    const formik = useFormikContext<TProofOfOwnershipFormValue>();
+    const { errors, setFieldValue, values } = formik;
+    const [showBrowseButton, setShowBrowseButton] = useState(
+        !values[paymentMethod]?.[methodId]?.files?.[subIndex as number]?.name
```

Comment:
What will happen if `subIndex` is string?

Reply:
I will change the type. It is a number